### ISSUE TITLE
create a visual queue indicating what blocked script is trying to be accessed

### ISSUE THREAD

**Issue by sanguis** [EFForg/privacybadger#320]
Sometimes I want to do something like (sign in via twitter) that privacy badger is blocking and I want to enable that one blocked  tracker. The hard part can be figuring out which service to unblock.

It would be if we could identify a script by its action, and then choose to enable it or not.

**Comment by cooperq** [comment]
I think that this is a good idea, though I'm having a little trouble visualizing how it would work. If you have any further thoughts on it I would love to hear them.

**Comment by sanguis** [comment]
in the ux it would be a "identify script by action" toggle (bool). 
When turned on when some one tried to access an action that is blocked (js event) either some sort of popup can be displayed or a tally next to the script within the plugin interface.

**Comment by Kinnera323** [comment]
@sanguis It is very difficult to visualize and gives a vague idea. Could you please elaborate it ?

**Comment by ghostwords** [comment]
We would like Privacy Badger to become an install-and-forget kind of extension. Anything that gets into customizing sliders needs to be clearly worth it benefit/complexity-wise. This proposal seems complex, while we already have a recommended alternative for when sites break, which is to use the "Disable Privacy Badger for This Site" button. Closing as "wontfix".

**Comment by ghostwords** [comment]
We would like to fix signing in with Twitter, etc. for all Privacy Badger users, however. Please subscribe to #137 for updates.
